src/languages/swift.js

```diff
@@ -362,6 +405,8 @@ export default function(hljs) {
           ...KEYWORD_MODES
         ]
       },
+      OPERATOR_DECL,
```

Comment:
Let's not abbreviate?  This is all gziped and minified later anyways.

Reply:
Fixed.